docs/docusaurus/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md

```diff
@@ -1,48 +1,158 @@
+
+:::tip What if I want to use the same Expectation more than once?
+
+GX takes into account certain parameters when determining if an Expectation is being added to the list or if an existing Expectation should be edited.  For example, if you are using an expectation such as `expect_column_*()` you would edit an existing Expectation by providing the same `column` parameter when repeating the previous steps, and different values for any other parameters.  If you used a different value for the `column` parameter, you will create an additional instance of the Expectation for the new `column` value, rather than overwrite the Expectation you defined with the first `column` value.
 
-### 2. Specify a Datasource (if multiple are available)
-<SpecifyADatasourceIfMultipleAreAvailable />
+:::
 
-### 3. Specify the name of your new Expectation Suite
-<SpecifyTheNameOfYourNewExpectationSuite />
+### 6. Save your Expectations for future use
 
-### 4. Continue the workflow within a Jupyter Notebook
-<ContinueTheWorkflowWithinAJupyterNotebook />
+The Expectations we have created so far are saved in an Expectation Suite on the Validator object.  Since Validators do not persist outside the current Python session these Expectations will not be kept if we stop now.  This can be ideal if we are using a Validator to do some quick data discovery, but in most cases we will want our newly created Expectation Suite to be available in the future.
 
-<Congrats />
+To keep our Expectations for future use, we will save them to our Data Context.  A Filesystem or Cloud Data Context will persist outside the current Python session, so saving the Expectation Suite in our Data Context's Expectations Store will ensure we can access it in the future:
 
-## Optional alternative Interactive Mode workflows
+```python name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_fluent.py save_expectation_suite"
+```
 
-### 1. (Optional) Edit an existing Expectation Suite in Interactive Mode
-<OptionalEditAnExistingExpectationSuiteInInteractiveMode />
+:::caution Ephemeral Data Contexts and persistence
 
-### 2. (Optional) Profile your data to generate Expectations, then edit them in Interactive Mode.
-<OptionalProfileYourDataToGenerateExpectationsThenEditThemInInteractiveMode />
+Ephemeral Data Contexts do not persist beyond the current Python session.  If you are working with an Ephemeral Data Context, you will need to convert it to a Filesystem Data Context using the Data Context's `convert_to_file_context()` method.  Otherwise, your saved configurations will not be available in future Python sessions as the Data Context itself will no longer be available.
```

Comment:
```suggestion
Ephemeral Data Contexts don't persist beyond the current Python session.  If you're working with an Ephemeral Data Context, you'll need to convert it to a Filesystem Data Context using the Data Context's `convert_to_file_context()` method.  Otherwise, your saved configurations won't be available in future Python sessions as the Data Context itself is no longer available.
```